core-bundle/src/Doctrine/Schema/DcaSchemaProvider.php

```diff
@@ -410,22 +410,20 @@ private function getDefaultIndexLength(Table $table): int
 
         $largePrefix = $this->doctrine
             ->getConnection()
-            ->query("SHOW VARIABLES LIKE 'innodb_large_prefix'")
-            ->fetch(\PDO::FETCH_OBJ)
+            ->fetchAssociative("SHOW VARIABLES LIKE 'innodb_large_prefix'")
```

Comment:
We could probably write this query like the following and also use `fetchOne()` instead:

```sql
SELECT VARIABLE_VALUE FROM information_schema.GLOBAL_VARIABLES WHERE VARIABLE_NAME = 'innodb_large_prefix';
```

This will return `false` if the variable does not exist.

Reply:
I am not sure whether this works on all platforms or if access to the `information_schema` database can be revoked, therefore I prefer to stick with the `SHOW VARIABLES` query.

Reply:
> In a MySQL 8.0, the `INFORMATION_SCHEMA` variable tables … are removed

https://dev.mysql.com/doc/refman/5.7/en/performance-schema-variable-table-migration.html

So I think we have to use `SHOW VARIABLES`